packages/cicero-core/test/template.js

```diff
@@ -65,12 +91,33 @@ async function writeZip(template){
 
     return await waitForEvent(output, 'close');
 }
-/* eslint-enable */
+/* eslint-disable */
 
 const options = { offline: true };
 
 describe('Template', () => {
 
+    describe('#signTemplate', () => {
+
+        it('should sign the cotent hash and timestamp string using the keystore', async() => {
+            const template = await Template.fromDirectory('./test/data/helloworldstate');
+            const timeStamp = Date.now();
+            const templateHash = template.getHash();
+            const signatureData = sign(templateHash, timeStamp, './test/data/keystore.p12', 'password');
```

Comment:
do those files in `./test/data` exist?

Reply:
I've stored these in `./test/data`  in my local repo, I'll push those. They did not get added to this commit.

Reply:
How were those files created? You probably want to avoid having your own private key there! Make sure those are test credentials 😉

Reply:
The `heloworldstate' is unzipped from the archive I downloaded from the template studio. The `keystore` is generated by an react app I made using node forge to generated some self-signed dummy keystores. Don't worry I don't have my private key in there ;). https://4fwcv.csb.app/